moncli/entities/column_value/simple.py

```diff
@@ -117,7 +117,46 @@ class PhoneValue(ComplexNullValue):
 
 class StatusValue(ComplexNullValue):
     """A status column value."""
+    native_type = str
+    allow_casts = (int, str)
 
+    def _convert(self, value):
+        settings = self.settings
+        labels = settings['labels']
+        index = str(value['index'])
+        value = labels[index]  
+        return value
+    
+    def _cast(self, value):
+        if isinstance(value,int):
+            index = str(value)
+            labels = self.settings['labels']
+            try:
+                value = labels[index]
+                return value
+            except KeyError:
+                raise ColumnValueError( 'invalid_status_index',self.id )
```

Comment:
I forgot to include this in the requirements docs.  The message should be the same as below:

    message: 'Cannot find a status with the following index "{}".format(value)